src/main/java/anab/Person/ManagePerson.java

```diff
@@ -2,8 +2,23 @@
 
 public class ManagePerson {
 
+    static void printeazaText(String name) {
+        System.out.println("Hello!" + name);
```

Comment:
Output here will be 'Hello!Ana'. Is this what you need?
If you need "Hello Ana!", you can change this line to 'System.out.println("Hello " + name + "!");'

Reply:
Format changed: firstName lastName.